src/Component.js

```diff
@@ -1,26 +1,25 @@
 import React from 'react';
 import {Motion} from 'react-motion';
+import PropTypes from 'prop-types';
 
+export class ReactMotionLoop extends React.Component {
+  constructor(props) {
+    super(props);
 
-export const ReactMotionLoop = React.createClass({
-  propTypes: {
-    styleFrom: React.PropTypes.object.isRequired,
-    styleTo: React.PropTypes.object.isRequired
-  },
+    this.onRest = this.onRest.bind(this);
```

Comment:
You can avoid `bind` by using lambdas, look at https://github.com/nkbt/react-works/blob/master/packages/swap/src/Component.js for example.